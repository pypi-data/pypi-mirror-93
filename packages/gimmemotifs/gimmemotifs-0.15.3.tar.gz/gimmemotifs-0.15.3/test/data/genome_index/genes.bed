chr2	38813	46588	uc010yim.2	0	-	41607	46385	0	2	2814,1149,	0,6626,
chr2	218135	256690	uc002qvv.3	0	-	218810	242800	0	12	866,57,79,169,129,113,74,60,65,114,111,484,	0,6728,11830,12887,14965,16024,24662,25367,29402,31595,34869,38071,
chr2	243502	262786	uc002qwc.1	0	-	243502	243502	0	6	60,65,114,111,618,156,	0,4035,6228,9502,16582,19128,
chr2	264868	278282	uc002qwh.3	0	+	264868	264868	0	7	139,74,29,114,62,106,1056,	0,6997,7168,7323,10271,12111,12358,
chr2	264868	278282	uc002qwf.3	0	+	264964	277304	0	6	139,74,114,62,106,1056,	0,6997,7168,10271,12111,12358,
