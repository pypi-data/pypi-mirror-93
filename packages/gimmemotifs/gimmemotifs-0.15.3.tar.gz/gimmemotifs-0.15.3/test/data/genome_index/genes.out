>uc002qvv.3
ATGGAGTTTCGTTCTTGCTGCCCAAGCTGGAGTGCAATGGCGCCATCTCG
GCTCACTGCAACCTCCACCTTCCAGGTTCAAGTGATTCTCCTGCCTCAGG
CTCCTGAGTAGCTGGGATTACAGGCGCACGTCACCACATCTGGCTAATTT
TTTGTATTTTTAGTAGAAATGGGGTTTCACCATGTTAGCCAGGCTGGTCT
TGAACTCCTGACCTCAGGTGATCCCCCCACCTCGGCCTCCCAAATTGCTG
GGATTACAGGTGTGAGCCACCGTGTCCGGCCATAAAACATTTTTTTACAA
AAGGCTTACAAAGGAATATTTATAGCAGTTTTATTTGTAATAGCCAAAAG
CTGGAAACAACCCAATTGTCTATCAACAGTGACTGGAAGGACAAATTGTG
CTATGTCCACACATTAGAATACCAGTTGGCAGTCAAAAAGGAAGAGCTGA
TCTGTGCAGCAATTTGAGTCTCAGTAATTTCATGTGAATAACCCTATACC
TTCCAATTTGAAATCAGAAGCAAAAAAGGCTGCCAAAATATTAAGAGAAT
TCACAGAAATAACTTCCAGAAATGGACCTGATAAGATCATTCCTGCTCAC
GTAATTGCGAAGGCTAAAGGCCTTGCAATTCTGTCTGTGATCAAAGCCGG
GTTCCTGGTGACTGCCAGAGGAGGCAGCGGGATTGTAGTGGCGCGCCTTC
CAGATGGAAAATGGTCTGCACCCTCAGCCATTGGGATAGCTGGCCTTGGT
GGAGGATTTGAAATAGGAATTGAGACACATTTACAGCTACATATTCCCTC
TGAGCATTGCCCTGGCTGCCTTCCATAAGTGTGGAGTGCCACTCCAGCTT
TCTTAGGTTGCCATCTGCATGACACATCTTTCTCCATCCTTTTACTGTCA
ATCTAATGGTATCAGACTTGGTGATAATTCTGAATTATGACCGTGCTGTA
GAAGCTTTTGCAAAAGGCGGAAATCTGACCCTCGGAGGGAACTTGACTGT
GGCGGTTGGGCCCTTGGGAAGGAACTTGGAAGGAAACGTGGCCCTGAGAA
GCTCCGCTGCCGTCTTCACGTACTGCAAGTCAAGGGGACTCTTTGCAGGC
GTGTCTTTAGAAGGGAGCTGTTTGATTGAAAGGAAAGAAACTAATAGAAA
ATTTTATTGTCAAGATATCCGAGCTTATGACATTTTATTTGGAGATACAC
CGCGGCCTGCTCAAGCCGAAGATCTTTATGAAATTCTTGATTCCTTTACT
GAAAAGTATGAAAATGAAGGACAACGAATCAATGCAAGAAAAGCAGCAAG
GGAGCAGAGGAAGTCTTCTGCTAAAGAATTACCTCCAAAGCCATTGTCAA
GACCACAGCAGTCATCTGCACCAGTCCAGCTGAACTCTGGCTCTCAAAGT
AACAGAAATGAATATAAGCTCTATCCTGGACTTTCCAGCTATCATGAGAG
AGTTGGCAATTTGAATCAACCCATAGAAGTGACAGCGCTGTATTCATTTG
AAGGACAGCAGCCTGGGGATTTGAATTTTCAAGCTGGAGACAGAATCACA
GTTATATCAAAAACAGATTCACATTTTGATTGGTGGGAAGGAAAACTTCG
AGGTCAAACTGGCATTTTTCCAGCCAACTACGTAACCATGAATTAAAGCG
TATACTATTTTCTTCTTTGAGAATTACAAAAAAATTATTTCTACACTGAC
AGGATTTACTAGTTAAGCAATGTTTAATATAAATTTTAAAAAACTTCTGT
TCTACAAAATTTCCATTCCGTATGTAAAAGATTTTGTTTTTCTATATAAA
AAGAGCTGACTGACATATCTTTAAATACTTTGTACTAACTTTATCACACT
TACTGTGTCATAGAATATCATACAGTTTATACGCTCATAGTTCTCTTGTG
AACACTTCAAACATCGCTAAGCATTTGATCTGGCCATGTATATGGTAGCT
GTGTTTTAATTTGAGAATCTTGAGGGTAGAGCCACAAATTTCAATTCTTA
CATTTCCATTTGCAAAGTGACTAGAGAAAAAGAAATCAGCTTAAATGAGG
TATTAAGTAATGTTTAGAGTCGTAGGTATTAACTAGAATATAAATCCTTA
GAAATTGTCTTTATACCTTCAAAAATTATACTATGCATTTATCATAGAAA
TGTGATTACAAAGAAGTCTGACTACCATGTCTTTAAACATATGGCATCTC
TCAACTTTTCTTCCTTATGGGGCTACATTTGTTCATTTCCAGCAGTAGCA
TAAACTTACGGTGACATGGTAGACTTGTCTCTAAATAAAATTTTTAAATG
TTTACTAAGTTAATATTTCAA
>uc002qwc.1
GAAAACTTTATTCTCAAGAGATTTCTCATTCCGCAGAAATAAGGATCTGG
CATGTCAACACAAACATAGGAAGTTGAGGCAGTCAAAGGGAAATAACTGC
TACCCTGACTCTGCTGAGAGCCTCACAAGTCCCTGTGCACGCCAGCTTGT
TTGCCAAAAAAGATGTTGCTGGTCCATTTGACGAAGAAGTCCAGAGCGGC
TTGTGACTTCAGTCTGGCTTTGCAGAGCCCAGGTGCTCTCTCCACCTTCT
GCCAGGTTCCCCTATGAAGGCTTTGCTCTCAGACTCTCCCCTTGTGGAGA
CAAAAATCCCACAGCAAGTGGAAACAGAAGTGCCTTGGAAAAAGTTATTC
TGGCAGCTCCTATACAAGTCCCCCATCAGATGCTGGTATTGGACCAGCTT
TGGTTATAAAGCCCTGCTGTGGCACCAAAAGGAATGTCGTCCTGATACAG
CAGCCTCGGGTCACCCAATCCACTCCAGATACTATGGCTGCAGCATCAAT
GTAAGCAGACAAGTTGTGCTGCCTCTGAAATAAAGTAGATGAAATATTCA
GCAGCAGAAAGAACAAGTTTCCACTCCAGTGAACTAATTCAAATGTATCC
AGCATCTAGATATATGTATAAATGTAATTAAAACATATTGAGCATCTAGA
GCATCTCTCCCAAGATAATTACCACCTGATAGTAATGGCTATAAAAGCTA
AAAGTGGAATCAGTCTACATGTTTCGCAATAGGAATCAATTGAAGCATAC
ATCAAAATGCAGACAGTAGTATTGTGAATAACCCTATACCTTCCAATTTG
AAATCAGAAGCAAAAAAGGCTGCCAAAATATTAAGAGAATTCACAGAAAT
AACTTCCAGAAATGGACCTGATAAGATCATTCCTGCTCACGTAATTGCGA
AGGCTAAAGGCCTTGCAATTCTGTCTGTGATCAAAGCCGGGTTCCTGGTG
ACTGCCAGAGGAGGCAGCGGGATTGTAGTGGCGCGCCTTCCAGATGGAAA
ATGGTCTGCACCCTCAGCCATTGGGATAGCTGGCCTTGGTGGAGGATTTG
AAATAGGAATTGAGACACATTTACAGCTACATATTCCCTCTGAGCATTGC
CCTGGCTGCCTTCCATAAGTGTGG
>uc002qwf.3
ACTGCGCAGGCGCGCGGGGCAAGAGGGTGGCAGTGCGCCTGCGCCGCGTC
GGCGTGCGGAACGCCGCGGTGTCTCGGCGCCTCTGCGCGCGGGAAGATGG
CGGAACAGGCTACCAAGTCCGTGCTGTTTGTGTGTCTGGGTAACATTTGT
CGATCACCCATTGCAGAAGCAGTTTTCAGGAAACTTGTAACCGATCAAAA
CATCTCAGAGAATTGGAGGGTAGACAGCGCGGCAACTTCCGGGTATGAGA
TAGGGAACCCCCCTGACTACCGAGGGCAGAGCTGCATGAAGAGGCACGGC
ATTCCCATGAGCCACGTTGCCCGGCAGATTACCAAAGAAGATTTTGCCAC
ATTTGATTATATACTATGTATGGATGAAAGCAATCTGAGAGATTTGAATA
GAAAAAGTAATCAAGTTAAAACCTGCAAAGCTAAAATTGAACTACTTGGG
AGCTATGATCCACAAAAACAACTTATTATTGAAGATCCCTATTATGGGAA
TGACTCTGACTTTGAGACGGTGTACCAGCAGTGTGTCAGGTGCTGCAGAG
CGTTCTTGGAGAAGGCCCACTGAGGCAGGTTCGTGCCCTGCTGCGGCCAG
CCTGACTAGACCCCACCCTGAGGTCCTGCATTTCTCAGTCGGTGTGTAAT
CACGTTCCAGGGCCCAAAGCCCAGCTCTTTGTTCAGTTGACTTACTGTTT
CTTACCTTAAAAAGTAATTGTAGATGGAAATCAGTTGTGTTTGGCAGGAG
AATCAATAAAAATCTTTGATTCAGACAGCTTATGGGGTATTTTAAGCATT
CTTAGACTAGTTGAACATCTCACTTTGCCCCAGTTACAAAAATAGTAGAA
CAAGCAACATAAAACAATGAAGGAAAACCTCACTTGAAGGCCCAGGTCAA
CATCTAAGCCTGTTGAGACTTAGATAATCGAGTCTACCTCTTCAGTAGGT
TTGTGTGGATGGCCTGGAGGGCAGGTGCCCTCTGCTCCCCAGTGCTACCT
CTCTCTTCCCTAGGGCCTTTTGTGGATTGACAGTAGTCCCCTCCGTAGGA
GCTCACAGTCTAGATTAGAAGTGTTTTAATTTCTACACACCCATAGTGCA
CACTTGTATATTGAAAAGATAGGGAAGAGAGAAACATTTATGGAATCAGT
CGTTGGCACCTTCAATACTTCATGATTTTTGTCGAGTTTACTTCATGAGG
AGGTCAGCCCATTGGCTCCCATCTGAACCACTTTGCCTCTGAAACTTAAT
TACATCCAGAAAGAAGGACACTTGTATGCTAGTCTATGGTCAGTTGAGGA
ATATGACTGTTTTTATATGCACATGTAACCCAAATGTCCAATATAAATTG
GCTTATTTTTTAAAATAATTTTAAAAGTTGGGAAAAGTGTTATTATTTGG
CATGCTTAAATATTGAATAAGTATTCTTCATCAGCATTTAATAAATGTAT
AGGCAGATGTAAGGTAATTTCTGTGTATTTTGAGATAATGTCAAAATCAT
GAATATTTCAAAATAAACTGGGGAGTTATAAAAATACAACTAGAGATATA
A
>uc002qwh.3
ACTGCGCAGGCGCGCGGGGCAAGAGGGTGGCAGTGCGCCTGCGCCGCGTC
GGCGTGCGGAACGCCGCGGTGTCTCGGCGCCTCTGCGCGCGGGAAGATGG
CGGAACAGGCTACCAAGTCCGTGCTGTTTGTGTGTCTGGGTAACATTTGT
CGATCACCCATTGCAGAAGCAGTTTTCAGGAAACTTGTAACCGATCAAAA
CATCTCAGAGAATTGGAGGGTAGACAGCGCGGCAACTTCCGGTGGGTCAT
TGACAGCGGTGCTGTTTCTGACTGGAACGTGGGCCGGTCCCCAGACCCAA
GAGCTGTGAGCTGCCTAAGAAATCATGGCATTCACACAGCCCATAAAGCA
AGACAGATTACCAAAGAAGATTTTGCCACATTTGATTATATACTATGTAT
GGATGAAAGCAATCTGAGAGATTTGAATAGAAAAAGTAATCAAGTTAAAA
CCTGCAAAGCTAAAATTGAACTACTTGGGAGCTATGATCCACAAAAACAA
CTTATTATTGAAGATCCCTATTATGGGAATGACTCTGACTTTGAGACGGT
GTACCAGCAGTGTGTCAGGTGCTGCAGAGCGTTCTTGGAGAAGGCCCACT
GAGGCAGGTTCGTGCCCTGCTGCGGCCAGCCTGACTAGACCCCACCCTGA
GGTCCTGCATTTCTCAGTCGGTGTGTAATCACGTTCCAGGGCCCAAAGCC
CAGCTCTTTGTTCAGTTGACTTACTGTTTCTTACCTTAAAAAGTAATTGT
AGATGGAAATCAGTTGTGTTTGGCAGGAGAATCAATAAAAATCTTTGATT
CAGACAGCTTATGGGGTATTTTAAGCATTCTTAGACTAGTTGAACATCTC
ACTTTGCCCCAGTTACAAAAATAGTAGAACAAGCAACATAAAACAATGAA
GGAAAACCTCACTTGAAGGCCCAGGTCAACATCTAAGCCTGTTGAGACTT
AGATAATCGAGTCTACCTCTTCAGTAGGTTTGTGTGGATGGCCTGGAGGG
CAGGTGCCCTCTGCTCCCCAGTGCTACCTCTCTCTTCCCTAGGGCCTTTT
GTGGATTGACAGTAGTCCCCTCCGTAGGAGCTCACAGTCTAGATTAGAAG
TGTTTTAATTTCTACACACCCATAGTGCACACTTGTATATTGAAAAGATA
GGGAAGAGAGAAACATTTATGGAATCAGTCGTTGGCACCTTCAATACTTC
ATGATTTTTGTCGAGTTTACTTCATGAGGAGGTCAGCCCATTGGCTCCCA
TCTGAACCACTTTGCCTCTGAAACTTAATTACATCCAGAAAGAAGGACAC
TTGTATGCTAGTCTATGGTCAGTTGAGGAATATGACTGTTTTTATATGCA
CATGTAACCCAAATGTCCAATATAAATTGGCTTATTTTTTAAAATAATTT
TAAAAGTTGGGAAAAGTGTTATTATTTGGCATGCTTAAATATTGAATAAG
TATTCTTCATCAGCATTTAATAAATGTATAGGCAGATGTAAGGTAATTTC
TGTGTATTTTGAGATAATGTCAAAATCATGAATATTTCAAAATAAACTGG
GGAGTTATAAAAATACAACTAGAGATATAA
>uc010yim.2
GGTCGGGAGGGGCGGGAACTGGGGAGTTTCGCCAATCGGTGCAGACTCCG
GTTCTCCCCAACCCCGGGCCTGGGCAGCTGGGCACTGGGGCGCCGTTCCC
CACCCGGCGCGGACTTCGGAACCGGCCCTGGGACTGACTGGCTCTACCAC
TCGAGCGCGTCTCCGCTGGACCCGGAACCCCGGTCGGTCCATTCCCCGCG
AAGATGCGCGCCCTGGCGGCCCTGAGCGCGCCCCCGAACGAGCGGCTCCT
TCCCCGGGACCCCGCGGCTACCCGGGACCCCGACGCCGCGCGGCCGGCGC
GCAGGAGCGCAGTGGAGAGGCTGGCGGCGGATCGCGCCAAGTATGTGCGG
GGTCGGCCGGGGACTGGCCGGGGCGTCGCTTCCGAGGGCAGCGGCCCGGG
GGCGATCAAGTGCCCGGGGAACGACCCTGGGCCCCCGGCCCGCGCCCCGG
CCCCGGTGGCGCGCAGGGCTATTGCGCGGAAGCCGTTGAGACCGGACTCG
CTGATCATCTACCGGCAGAAATGCGAATTCGTGCGAGGATCGGGCGCCGA
CGGCCCCAGGGCAAGCCTGGTGAAGAAGCTCTTCCAGGGGCCGGGTAAGG
ACAAGGCGCCGGTGCCCCGGACGGGAGACGAGGGCAAGGCCGGGAACCCG
GAGACGGTCCCGACCACTCCTGGCCCCGCGGCGGACCCCGCAATCCCCGA
GACCCCAGCCCCAGCCGCGCGGTCCGCGGCGCCCTCCAGTGTCCCGGCCG
CGCCCCCTGGGCCAGAGCCGCGGGTGGTGAGGCGTCGGGGGCTGCAGCGC
TCACAGTCGGACCTCAGCTCCCGCTATTCCGCTGCCTTGGCCGAGTCTGA
CACCTTCTTCCAGTACTGCGGCCTGGACCCCGAGGTGGTGGAGGCCCTGG
GGAGGGAGAACTTCACCGCGGGGTCGGACTGTGTGACCCTCAAGGTGCGC
AGCGTGAGCGTCGCCACCTCCGGCAGCGGCTTCTCCCGGCACAGCGGCGG
CGACGACGAGGGGCTGCAGGAGGAGGAGCTGATAGAGCAGGTGCCCAGCA
CCACTTCGGTCATCGAGAGGAACGCCCGCATCATCAAGTGGCTGTACACC
TGTAAGAAGGCCAAGGAGACCCCCAGCCAGGAGCAGAGCCGGACCCGAGG
AAGCAAACCTTCCCGATGATGAAGACTTCTTGAAGGATTTCATGGGTGAT
TTGGGATCCCAGGACCATTTGGCTAGTGTGCCTAGGTGACCACATGATTG
CTGTTTTACCAGGAATGCAGCATCCCATTGACAAAACAAGTGCTCTGAGA
AGGTTTAAAATACTACAGAGAATATGGGAACACAGACCTTGAAATTTAGC
TGAGTTGTAACAGCTGAAACTCCAAGAGGTGTCTTCCTTGTTTGAGGTGA
AACTAGTGTTGCTTCCAGAGGGCAGCTGGAAACCGTAAAGCTGTTTGGAA
ATCTTTTTGACTGACTTGCTGACAAAGAGGTACTGTGATGCATTTTAACA
ATATCTAAGTTGATTTTTTTTTAAATCAAGGAAAATAAAAACCAAGCATG
AATGCTATGGTATGTGCCCCTTTTGACCATCCTGGGCTGATTAACATCAT
TTAAATCAAAGTAATCATAAAAAGGCATATTCTACTTCAATTATGTGGTC
AAATAAGAGTAAACACACACACTCACACATGCTGACCCCAATTGCCAGAG
CATTACTGCACTATAAATTACGGTTAATTCCCAAATTATACTACTGTTTA
TCTTATTTAACAAGTCAGAAAGCACTTTTAAAATAACTTGAGGGCTACAA
GGTCATTCTATTAATGTCATTCTCCATTCGGGTTGTAGGCATGTGGAAGT
ACCCATTAAAAGATAAGTTAGAGTTTAAATACTGATAAACAAAACCTTTT
ATTGCAACTGGACAGTTTCTGGAGAGTTAGCGGAAGAATCTTGGAGTTTC
CTTTGGTCAGATGAATACAACATTTCACTTTTGCAGCACTATTTAGAATG
TACTCCATGGTTCTCTTGTTCCCAACTTCCAAAAAGAACAGAAAACTTTG
GTTTACACAGAACACGGGCATCTGAGGCAGGACCTCTTCCCTGCCCTTTG
ATCTGACTCACACCTCCACATATGACGTAATCAACCCAAATTTGACACCA
ATTCACTCTTTTCTGCAAAGGGCATATTTTGAAACAAGGGACAGCCTGAG
GGCGGCTATAATGAGAATGTTCATGGGGGTTACTGGGTCCCTAATTCTGA
ACTTGCTTATGACACCCAGAGTGAATAGATTCAGATTCAGAACCTTCTGA
GAAATAACCCAAAGAAAATTTGTTACCCAGCCAATTCTTCGAAAGCTTAA
TATCAAAATATATCTTTTCAAGAAGAAAATCGTTAGAGAGAAGAATGTGG
AGGGGAGAGAAATGGGTTTCTCATTGATATGATATTTTGTTAACCATTTC
ATTTTGAATTATTCAAGTTTTGGTTAATATTGTATTCTTTTTTCGTAACT
ATTTTACCGTGAGAGTAGGTCATTGGGTTACTTAGATATTTATTTTTACA
CAGTTATTAGTCTTCAGATAGTTTTATTTTACTTCATATGATTTTAGTTT
TTGTCAGTATAATTTTAAATCATGTTTTTCTTGGTCATCTCTTTGTGTAT
ATTGTGTAATTGGATTTTCATTGACTGCAAGTGGAGTGTTTGCCACTCAA
TTCAGTACTCAGTACTATGGTGACTTGTTTTCAAATAAGTCTCAGATACA
CATTTAGGGAGCCTTTGCTGGCCGAATATAGACTCTGTCAGGACAGCAGG
TCCCCTGATCTAAGAATTTTCCCCAATGGTTGCTCTAAAAATGCTGCTAT
TTTGCTGTTCACTGTATTGCACTTAGTTAAAAAGAAGATAATGTGAAAGA
TGAGAGCAGTTTTTTAAAGGATCTTTTCATATACCCAATTCCCTTATTTT
CAGATGTCCCATCAATTTTAGATATGAAAGCTTTAAGTAAAAGTGTGTAT
GCCTTTCTACTGTCAGAACAGGATGGATGCAGCCTGGGTCAGATTTATTT
AAGATAAAAATCATGCAGACTCATCATTCATATCATAGGTGAAAAATGTA
AAAACCAAATGGTTTCCACTAAAGCCACCAAGATCTTTTAGAAATGTTTG
CACCTTTGGTGGTGGCACAGGAAAAGAGAAGAATTCAGCTGGAGTGAATT
CTAGAAGTAGATATCAGAAACGGGGCATGAAGAACAGGGGAACTGGGTGG
CATCAGACTCCTAAAGAAGTGAGTTAATTTTCCTTCCCTTCCATTCAGAT
TCATGCCACAGCTCCATATCTTGAGTATGTGTAAGAGGTGAGTTCCTTCT
TCAGCCAGGGGCGGTGGCTCATGCCTTTAATCCCAATGCTTTGGGAGGCC
AAGGTGGGAGGATCACTTGTGCCTTGGGGTTCAAGGTTGCAGTGAACCAT
GATTGCACCACTGCACTCCAGCCTGAGTGACAGAGCAAGACCCTGTCTCT
AAAAATATATATAAAAAGTAAAACTAAAGAACTTCTTGCCTAAACCTGAA
TTACCGCAATTTGCTGAGTGACTTTGAGAAAAATCAGACTGTTTAGTTCA
GTCGGGATGAAAAGCTTGCGATTGCTTCCCACAAGAATGGGCAATAGTGA
CGGCTGCAAGGTACTTTTATTTGTTCATGAAAGAACGACAATTTTTCAAA
ATGTAATTAAACATAATAGAATGTTTTAAACTACTGGGCACTGAAACTGG
AAGAAAAAGGAGGCTTTATTGAACATTCCCCTTTTTCAGTTGGTTCAAAG
TTCAGCACTGTGGTTATCATTGGTGATGCCAGAAAACATTAGTAGACTTA
GACAATTGCTATGGCAGTTTCTAAACAGAGCTTTTTCTATACACTATTTG
CAACTGGAGTGCAATATTGTATATTCTGTGTTAAAGAAATAAAGTATTTT
TATCATTTATTAA
